spec/factories.rb

```diff
@@ -1,36 +1,42 @@
-FactoryGirl.define do
-  factory :user do
-    name "Example user"
-    sequence(:email) {|n| "user#{n}@example.com"}
-    password "somepassword"
-  end
+FactoryGirl.define do  factory :review do
```

Comment:
Unnecessary spacing detected.<br>Block body expression is on the same line as the block start.